nmostesting/NMOSTesting.py

```diff
@@ -431,6 +432,33 @@
         }],
         "class": BCP0060102Test.BCP0060102Test
     },
+    "BCP-004-02": {
+        "name": "Sender Capabilities",

```

Comment:
Make name consistent with other test suites
```suggestion
        "name": "BCP-004-02 Sender Capabilities",
```